src/app/games/diam/diam.component.ts

```diff
@@ -50,7 +50,7 @@ interface LastMoved {
     templateUrl: './diam.component.html',
     styleUrls: ['../../components/game-components/game-component/game-component.scss'],
 })
-export class DiamComponent extends GameComponent<DiamRules, DiamMove, DiamState> {
+export class DiamComponent extends GameComponent<DiamRules, DiamMove, DiamState> implements OnInit {
```

Comment:
Does that change things ?
Btw: perhaps typing ngOnInit, GitHub Actions complain :p

Reply:
ngOnInit typé
le `implements OnInit` est pas strictement nécessaire mais préférable car on dépend du ngOnInit ? (il semble être appelé même si on déclare par `implements OnInit`, mais je vois pas de raison de pas le faire ?)

Reply:
Vrai, c'est p-e plus propre. Tu peux résoudre!